lzy/allocator/src/main/java/ai/lzy/allocator/alloc/impl/kuber/KuberMountHolderManager.java

```diff
@@ -212,6 +201,7 @@ public static VolumeMount prepareVolumeMount(ServiceConfig.MountConfig mountConf
 
     @NotNull
     public static VolumeRequest createHostPathVolume(ServiceConfig.MountConfig mountConfig) {
+        // TODO: random UUID ?
         return new VolumeRequest("host-path-volume-" + UUID.randomUUID(),
```

Comment:
@Hoxo , why do we use random here?
won't it break the repeatability (idempotency) of the operation step?

Reply:
No, it won't. Random here is just for precaution, because it's safer it terms of names collision. And this volume is created once per pod and its id used only in pod, so we don't have to worry about idempotency.